helix-core/src/main/java/org/apache/helix/PropertyKey.java

```diff
@@ -241,6 +243,13 @@ public PropertyKey clusterConfig() {
           _clusterName, ConfigScopeProperty.CLUSTER.toString(), _clusterName);
     }
 
+    /**
+     * Get a property key associated with this cluster status
+     * @return {@link PropertyKey}
+     */
+    public PropertyKey clusterStatus() {
+      return new PropertyKey(STATUS, ClusterStatus.class, _clusterName);
+    }
```

Comment:
Question: from this property key, it looks like on the top of root path?

Reply:
It's `/{clusterName}/STATUS/CLUSTER/{clusterName}`